content/v1.13-docs/contributing/README.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Contributing
+description: 'cert-manager contributing guide: Get involved!'
+---
+
+## Great to See You!
+
```

Comment:
suggestion: we don't usually include the "contributing" section for older release docs